### PR TITLE
[10.0][T69][ADD/IMP]: Check in/Check out attendance entry in auditlog based on tz

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
- TASK #[69](https://www.quartile.co/web#id=69&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

**Comment by YogeshMahera-SerpentCS** [comment]
Hi @TimLai125 can you please review it?

**Review by yostashiro — CHANGES_REQUESTED** [review]
@YogeshMahera-SerpentCS Can we use context_timestamp() for the main part of your logic?

Please also update the description of the manifest file to add note to pay attention to the superuser's timezone setting.

File: auditlog_ext_nrq/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,11 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     'name': "Audit Log Extension NRQ",
-    'version': "10.0.1.0.0",
+    'version': "10.0.1.0.1",
+    "description": """
```
Comment (TimLai125) [inline auditlog_ext_nrq/__manifest__.py]: description is deprecated, we should put it to the readme

File: auditlog_ext_nrq/models/auditlog_log.py [hunk 2]
```diff
@@ -31,3 +30,34 @@ def _get_log_category(self):
                 log.log_category = 'create'
             elif log.method == 'unlink':
                 log.log_category = 'unlink'
+
+
+class AuditlogLogLine(models.Model):
```
Comment (TimLai125) [inline auditlog_ext_nrq/models/auditlog_log.py]: We should create a new python for this class, i.e. auditlog_log_line

**Review by TimLai125 — CHANGES_REQUESTED** [review]

**Comment by TimLai125** [comment]
Please help to update the license to LGPL

File: auditlog_ext_nrq/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+This modules does the following:
+  * Allows to user can check the audit log entry based on user timezone only for datetime field.
```
Comment (TimLai125) [inline auditlog_ext_nrq/readme/DESCRIPTION.rst:2]: Please re-generate the readme documents (index.html etc.)

**Review by TimLai125 — CHANGES_REQUESTED** [review]

**Review by TimLai125 — APPROVED** [review]

File: auditlog_ext_nrq/models/auditlog_log.py [hunk 4]
```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 # Copyright 2019 Quartile Limited
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 from odoo import api, fields, models
```
Comment (TimLai125) [inline auditlog_ext_nrq/models/auditlog_log.py:5]: Add back the empty line above the import

File: auditlog_ext_nrq/models/auditlog_log_line.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import api, fields, models
```
Comment (TimLai125) [inline auditlog_ext_nrq/models/auditlog_log_line.py:5]: Add an empty line above the import

File: auditlog_ext_nrq/models/auditlog_log_line.py [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Quartile Limited
```
Comment (TimLai125) [inline auditlog_ext_nrq/models/auditlog_log_line.py]: 2020

File: auditlog_ext_nrq/models/auditlog_log.py [hunk 7]
```diff
@@ -1,7 +1,6 @@
 # -*- coding: utf-8 -*-
 # Copyright 2019 Quartile Limited
```
Comment (TimLai125) [inline auditlog_ext_nrq/models/auditlog_log.py]: 2019-2020

File: auditlog_ext_nrq/__manifest__.py [hunk 8]
```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
 # Copyright 2019 Quartile Limited
```
Comment (TimLai125) [inline auditlog_ext_nrq/__manifest__.py]: 2019-2020

**Review by TimLai125 — CHANGES_REQUESTED** [review]
Update to "Request Changes"

**Comment by YogeshMahera-SerpentCS** [comment]
Hi @TimLai125 amended changes!

**Pull request merged**